src/ui/components/LedgerViewer.js

```diff
@@ -216,9 +223,47 @@ const LedgerEventRow = React.memo(
                 id={action.identity.id}
                 name={action.identity.name}
               />
-              <Chip label={action.identity.subtype} size="small" />
+              <Chip
+                className={classes.chip}
+                label={action.identity.subtype}
+                size="small"
+              />
             </>
           );
+        case "ADD_ALIAS":
+          try {
+            const account = ledger.account(action.identityId);
+            let alias = action.alias.description;
+            // description has format: channel/[handle](optional url). Parse out channel & handle
+            const descriptionParts = alias.match(/^([^/]*)\/\[([^\]]*)\]/);
+
+            if (descriptionParts) {
+              const [, channel, handle] = descriptionParts;
+              alias = `${channel}/${handle}`;
+            }
+
+            return (
+              <>
+                <IdentityDetails
+                  id={action.identityId}
+                  name={`${account.identity.name} ⇐ ${alias}`}
+                />
+                <Chip
+                  className={classes.chip}
+                  label={account.identity.subtype}
+                  size="small"
+                />
+              </>
+            );
+          } catch (e) {
+            console.warn("Unable to find account for action: ", action);
+            return (
+              <IdentityDetails
+                id={action.identityId}
+                name="[Unknown Account]"
+              />
+            );
+          }
```

Comment:
Should extract this logic into its own component since there's a decent amount of logic here.